data/avro-schema/src/main/resources/avro/net/corda/data/virtualnode/VirtualNodeUpdateRequest.avsc

```diff
@@ -0,0 +1,16 @@
+{
+  "type": "record",
+  "name": "VirtualNodeCreationRequest",
+  "namespace": "net.corda.data.virtualnode",
+  "fields": [
+    {
+      "name": "x500Name",
+      "type": "string"
+    },
+    {
+      "name": "updateActor",
+      "type": "string",
+      "doc": "ID of RPC user that requested the virtual node creation."
+    }
```

Comment:
You will need version as well in the request as you need to do a version match with the existing concurrency control.

Have a look at Configuration (https://github.com/corda/corda-api/blob/release/os/5.0/data/avro-schema/src/main/resources/avro/net/corda/data/config/ConfigurationManagementRequest.avsc) as an example.